r/tests/testthat/test-dataset-write.R

```diff
@@ -469,9 +501,6 @@ test_that("Dataset writing: unsupported features/input validation", {
   expect_error(
     write_dataset(ds, tempfile(), basename_template = "something_without_i")
   )
-  expect_error(
-    write_dataset(ds, tempfile(), basename_template = NULL)
-  )
```

Comment:
I think these lines should not be removed.

Reply:
thank you.